acrg/obs/process_gcwerks_parameters.json

```diff
@@ -25,12 +25,12 @@
             }
         },
         "directory" : {
-            "GCMD":   "/user/work/chxmr/shared/obs_raw/AGAGE_GCWerks/data/",
-            "GCMS":   "/user/work/chxmr/shared/obs_raw/AGAGE_GCWerks/data-gcms/",
-            "medusa": "/user/work/chxmr/shared/obs_raw/AGAGE_GCWerks/data-gcms/"
+            "GCMD":   "/group/chemistry/acrg/obs_raw/AGAGE_GCWerks/data/",
+            "GCMS":   "/group/chemistry/acrg/obs_raw/AGAGE_GCWerks/data-gcms/",
+            "medusa": "/group/chemistry/acrg/obs_raw/AGAGE_GCWerks/data-gcms/"
         },
         "sampling_period": {
-            "GCMD": 75,
+            "GCMD": 120,
```

Comment:
Is this new sampling period correct for all data past and present? Or is this a new sampling period for more recent data?